scripts/dts/extract/reg.py

```diff
@@ -6,6 +6,7 @@
 
 from extract.globals import *
 from extract.directive import DTDirective
+from extract.edts import *
```

Comment:
We need to add 'ranges' support.  Its not specifically related to eDTS, but will impact it.